core/cli/main.py

```
@@ -919,6 +953,46 @@ def database_update(
+    """
+    start_time = time.time()
+
+    try:
+        from core.cli.commands.database import export_collection
+
+        response = export_collection(collection, output, start_time, limit=limit)
+        # Only print the response JSON if writing to a file (not stdout)
+        if output:
+            print_response(response)
+        if not response.success:
+            raise typer.Exit(1) from None
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)